### PR TITLE
Create SQL Server and Oracle network policies for database connectivity.

### PR DESCRIPTION
**Pull request by bryanpaget**
Jira: https://jirab.statcan.ca/browse/BTIS-432

Add network policies for all users for sql server and oracle.

### CODE REVIEW

File: cmd/network.go [hunk 1]
```diff
@@ -275,9 +277,68 @@ func generateNetworkPolicies(profile *kubeflowv1.Profile) []*networkingv1.Networ
...
+	policies = append(policies, &networkingv1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "allow-sql-ingress",
+			Namespace: profile.Name,
+			OwnerReferences: []metav1.OwnerReference{
+				*metav1.NewControllerRef(profile, kubeflowv1.SchemeGroupVersion.WithKind("Profile")),
+			},
+		},
+		Spec: networkingv1.NetworkPolicySpec{
+			PodSelector: metav1.LabelSelector{}, // Apply to all pods in the namespace
```
Comment (Souheil-Yazji) [inline cmd/network.go:290]: should only apply to notebooks.

**Review by Souheil-Yazji — CHANGES_REQUESTED** [review]

File: cmd/network.go (hunk 1 above)
Comment (bryanpaget) [reply cmd/network.go]: I defined a PodSelector near the top of the file and now I've referenced the PodSelector in each relevant policy.

**Comment by bryanpaget** [comment]
This was tested on dev, the results can be seen here: https://jirab.statcan.ca/browse/BTIS-432

**Review by Jose-Matsuda — APPROVED** [review]

**Comment by bryanpaget** [comment]
Egress from notebook pods was added for oracle and sql ports, ingress was unnecessary.

**Pull request merged**